lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts

```diff
@@ -16,7 +16,8 @@
  */
 
 import { FormFields } from '../formFields';
-import { by, Locator } from 'protractor';
+import { by, element, Locator } from 'protractor';
+import { BrowserVisibility } from '@alfresco/adf-testing';
```

Comment:
this import create a circular dependency please see the other file inside this module how they import the BrowserVisibility.

Reply:
changed the import line for BrowserVisibility